tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -2138,14 +2138,14 @@ Status Remapper::Optimize(Cluster* cluster, const GrapplerItem& item,
           continue;
         }
 
-        // Remap Conv2D+BiasAdd+Add into the _FusedConv2D.
-        if (FindContractionWithBiasAddAndAdd(ctx, i,
-                                             &contract_with_bias_and_add)) {
-          TF_RETURN_IF_ERROR(
-              AddFusedContractionNode(&ctx, contract_with_bias_and_add,
-                                      &invalidated_nodes, &nodes_to_delete));
-          continue;
-        }
+        // // Remap Conv2D+BiasAdd+Add into the _FusedConv2D.
```

Comment:
Why to remove those?

Reply:
the _MklFusedMatMul contains some extra ops pattern, matmul+biasadd+addn/Gelu, which is not implemented at the origin kernel. If we just remove the name in the list directly, TensorFlow will encounter an unsupported ops error. I have added condition check to avoid this problem in the latest update. pls review it again, thx~